comp/core/gui/guiimpl/gui.go

```diff
@@ -202,7 +202,10 @@ func generateIndex(w http.ResponseWriter, _ *http.Request) {
 		return
 	}
 
-	e = t.Execute(w, map[string]bool{"restartEnabled": restartEnabled()})
+	e = t.Execute(w, map[string]any{
+		"restartEnabled":    restartEnabled(),
+		"logginInstruction": logginInstructions(),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** i guess it's a typo, should be `loginInstructions` ?